scripts/devnet-run.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# This script starts devnet in background and checks until it is responsive.
+# Outputs the PID of the bg process.
+
+set -eu
+
+HOST=127.0.0.1
+PORT=5050
+
+if [[ -n "${STARKNET_HARDHAT_DEV:-}" ]]; then
+	echo "Running dockerized Devnet..."
+	# Get default from config.json file
+	if [[ -e "./config.json" ]]; then
```

Comment:
Why first check in current dir and then default to parent?

Reply:
We might or might not be inside `example` repo at this point.

Reply:
You can add these last two sentences as a comment in the script (also, when just updating the comments, feel free to use `[skip ci]` in the commit message)